packages/wallet/test/services/RewardsTracker.test.ts

```diff
@@ -16,8 +16,8 @@ describe('createRewardsTracker', () => {
             body: {
               withdrawals: [
                 {
-                  address,
-                  quantity: 100n
+                  quantity: 100n,
+                  stakeAddress: address
```

Comment:
On line 9:

``` TypeScript
    const address = keyManager.deriveAddress(0, 0);
```
It should be:
``` TypeScript
    const stakeAddress = keyManager.stakeKey.to_bech32();
```

Reply:
Good catch. This exposed another issue that rewards provider didn't even need addreses[] that it used for calling `walletProvider.utxoDelegationAndRewards`